src/Lykke.AlgoStore.Services/AlgoInstanceHistoryService.cs

```diff
@@ -53,6 +53,25 @@ public async Task<AlgoInstanceTradeResponse> GetTradesAsync(string instanceId, s
             return result;
         }
 
+        public async Task<IEnumerable<QuoteChartingUpdate>> GetQuotesAsync(string instanceId, string asetPair, DateTime fromMoment, DateTime toMoment, bool? isBuy, string clientId,  ModelStateDictionary errorsDictionary)
+        {
+            var authToken = await GetAuthToken(instanceId, clientId, errorsDictionary);
+            if (String.IsNullOrWhiteSpace(authToken))
+                return null;
+
+            var quotes = await _historyService.GetQuotes(fromMoment, toMoment, asetPair, instanceId, authToken, isBuy);
+
+            if (quotes == null)
+            {
+                errorsDictionary.AddModelError("ServiceError", "Unknown");
+                return null;
+            }
+
+            var result = quotes.Select(AutoMapper.Mapper.Map<Lykke.AlgoStore.Algo.Charting.QuoteChartingUpdate>);
```

Comment:
could be something like:
var result = AutoMapper.Mapper.Map<IEnumerable<QuoteChartingUpdate>>(quotes);

Reply:
yep, it its valid suggestion. Both will actually achieve the same thing though. 
quotes.Select returns IEnumerable of what in is in the query. My version is just a 'short' hand of 
quotes.Select(q=>AutoMapper.Mapper.Map<Lykke.AlgoStore.Algo.Charting.QuoteChartingUpdate>(q));  , generated by Resharper.  I agree that what you suggest is probably easier to read :)